pkg/elasticsearch/elasticsearch.go

```diff
@@ -154,3 +156,18 @@ func transformSearch(orgId int64, search []byte) ([]byte, error) {
 
 	return json.Marshal(s)
 }
+
+func ElasticsearchProxy(c *middleware.Context) {
```

Comment:
is this function used anywhere? i can't find any references to it

Reply:
nope.  removed it